CMakeLists.txt

```diff
@@ -29,7 +30,11 @@ else()
 endif()
 
 if(IS_CI_BUILD)
-    add_compile_options(-DCI_BUILD=1)
+    target_compile_options(-DCI_BUILD=1)
+endif()
+
+if(WITH_CLIPPER)
+    target_compile_options(-D__CLIPPER__)
```

Comment:
Can we instead use something like `MBGL_USE_CLIPPER` to be in line with other defines?